src/frontend/web_application/src/scenes/DiscussionList/components/DiscussionItemActionsContainer/presenter.jsx

```diff
@@ -66,8 +66,13 @@ class DiscussionItemActionsContainer extends Component {
         >
           {__('discussion-item-actions.action.more')}
         </DropdownController>
-        <Dropdown id={this.dropdownId} position="bottom" closeOnClick>
-          <VerticalMenu>
+        <Dropdown
```

Comment:
mmm I thought the good way to call Raw was via `raw` prop:
```
<Dropdown
  id={this.dropdownId} 
  position="bottom" 
  closeOnClick 
  raw
  className="m-discussion-item-actions-container__dropdown"
>
```
(the other way doesn't seem to work)

but! the only difference I see between Dropdown and Raw is the class `.m-dropdown`, which is responsible for the foundation Dropdown styles, including position and visibility (via `@include dropdown-container;`). 

So imo:
- className `m-dropdown` should alway be applied to Dropdown comp. even the raw one;
- `.m-dropdown` should only contain foundation-dropdown styles (no border, no background, no shadow)
- there should be modifiers for stylized Dropdown (ex: `--dark` for current user menu and application Switcher) 

What do you think of that?